cpu/ppc64_cpu_test.py

```diff
@@ -76,7 +80,8 @@ def test(self):
             process.system_output("ppc64_cpu --info")
             self.smt()
             self.core()
-            self.subcore()
+            if self.smt_subcores == 1:
```

Comment:
Since this code is inside a loop, a flag will help save multiple calls to os.path.exists